copilot/fastapi/tests/conftest.py

```
@@ -0,0 +1,62 @@
+        mock_token.return_value = "test-argo-cd-token"
+        yield mock_token
+
+@pytest.fixture
+def mock_git_commands():
+    """Fixture to mock Git subprocess commands"""
+    with patch("subprocess.run") as mock_run:
+        mock_run.return_value = MagicMock(returncode=0)
+        yield mock_run
+
+@pytest.fixture
+def mock_yaml_operations():
```

Ruff (0.8.2)

Missing return type annotation for public function `mock_yaml_operations`

(ANN201)